project_from_quotation/wizards/project_from_quotation_views.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="project_from_quotation_action" model="ir.actions.act_window">
+        <field name="name">Create Quotation Project</field>
+        <field name="res_model">project.from.quotation</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+    </record>
+
+    <record id="project_from_quotation_view_form" model="ir.ui.view">
+        <field name="name">project.from.quotation.view.form</field>
+        <field name="model">project.from.quotation</field>
+        <field name="arch" type="xml">
+            <form string="">
+                <sheet>
+                    <field name="sale_order_id" invisible="1" />
+                    <group>
+                        <group>
+                            <field name="mode" />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page name="order_lines" string="Order Lines">
+                            <field name="order_line_ids">
+                                <tree>
```

Comment:
No need to show these fields
![image](https://github.com/OCA/sale-workflow/assets/2234999/64f311f8-0172-43bf-b6ee-adc9be97d9dc)